packages/esbuild/esbuild.0.14.34/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+synopsis: "Esbuild prebuild command line on opam"
+description: """Esbuild prebuild command line on opam."""
+maintainer: ["Thibaut Mattio"]
+authors: ["Thibaut Mattio"]
+license: "ISC"
+homepage: "https://github.com/tmattio/opam-esbuild"
+doc: "https://tmattio.github.io/opam-esbuild/"
+bug-reports: "https://github.com/tmattio/opam-esbuild/issues"
+dev-repo: "git+https://github.com/tmattio/opam-esbuild.git"
+install: [ 
+  "cp" 
```

Comment:
You should use the `.install` file. This cannot work due to the sandbox:
```
[ERROR] The installation of esbuild failed at "cp esbuild-linux-x64 /home/opam/.opam/4.02/bin/esbuild".
- cp: cannot stat 'esbuild-linux-x64': No such file or directory

#=== ERROR while installing esbuild.0.14.34 ===================================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.02.3 | pinned(https://github.com/tmattio/opam-esbuild/releases/download/0.14.34/esbuild-0.14.34.tbz)
# path                 ~/.opam/4.02/.opam-switch/build/esbuild.0.14.34
# command              ~/.opam/opam-init/hooks/sandbox.sh install cp esbuild-linux-x64 /home/opam/.opam/4.02/bin/esbuild
# exit-code            1
# env-file             ~/.opam/log/esbuild-32-f3df4f.env
# output-file          ~/.opam/log/esbuild-32-f3df4f.out
### output ###
# cp: cannot stat 'esbuild-linux-x64': No such file or directory
```
It is also a bit weird. Normally opam is to compile sources and install, not to copy pre-built binaries. Would it be possible to have esbuild work like the other opam packages?